src/lib/drivers/driver.js

```diff
@@ -159,7 +159,9 @@ class DriverBase {
 
   gotoURL(url, options) {
     const waitForLoad = (options && options.waitForLoad) || false;
+    const disableJavaScript = (options && options.disableJavaScript) || false;
```

Comment:
can you add a comment in scheduler.js to clarify that the 2nd reload is with javascript disabled?  
later we might even rename these method names to keep it super clear.